### PR TITLE
Weekly View Fix and Spacing adjustments

### LINKED ISSUE #141: Index page mobile responsive

### LINKED ISSUE #374: Text doesn't wrap on weekly view
**Describe the bug**

**To Reproduce**
1. Go to Daily View
2. Make an entry with a word at least as long as "misunderstanding"
3. Go to Weekly View

**Expected behavior**

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser: Brave
 - Version 1.24.86
 - 

**Additional context**
This is a quick fix that can be implemented by changing the break type.

### PR DESCRIPTION
**Pull request by klbaker**

### CODE REVIEW

File: source/javascript/src/utils/Router.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const prod = true
+const prod = false
```
Comment (will-chung) [inline source/javascript/src/utils/Router.js:1]: I'm not sure whether this should be kept `true` or `false`. I think when testing locally it should be `false`, but when pushing onto the main branch it should be `true`? @yuzilyu @NoahT

Comment (yuzilyu) [reply source/javascript/src/utils/Router.js:1]: yeah if it's false the actual website's routing won't work

Comment (yuzilyu) [reply source/javascript/src/utils/Router.js:1]: I think we could merge it and change this line of code directly on github

**Review by yuzilyu — APPROVED** [review]
I am going to change one line for cd

**Pull request merged**